app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -12,9 +12,7 @@
           </div>
           <div class="lh-1">
             <h2 class="mb-0"><%= @pet.name %></h2>
-            <% if @pet.has_adoption_pending? %>
-              <p class="mb-0 d-block"><%= t('general.adoption_pending') %></p>
```

Comment:
@jadekstewart3 did you intend to diverge from the en.yml translation here?

Reply:
This is fixed :)